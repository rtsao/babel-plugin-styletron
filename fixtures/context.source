const H1 = (_, {styletron}) => <h1 style={{color: 'red'}}/>;
