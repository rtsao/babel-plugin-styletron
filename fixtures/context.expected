const H1 = (_, { styletron }) => <h1 className={Styletron.injectStyle(styletron, { color: 'red' })} />;
